metrics/web/lib/project_groups/presentation/widgets/project_group_card.dart

```diff
@@ -59,38 +59,55 @@ class _ProjectGroupCardState extends State<ProjectGroupCard> {
                   Row(
                     mainAxisAlignment: MainAxisAlignment.spaceBetween,
                     children: <Widget>[
-                      IconLabelButton(
-                        onPressed: () => _showProjectGroupDialog(context),
-                        borderRadius: _buttonBorderRadius,
-                        iconPadding: _buttonIconPadding,
-                        icon: Image.network(
-                          'icons/edit.svg',
-                          width: _iconBoxSide,
-                          height: _iconBoxSide,
-                          fit: BoxFit.contain,
-                          color: theme.primaryColor,
-                        ),
-                        label: CommonStrings.edit,
-                        labelStyle: TextStyle(
-                          color: theme.primaryColor,
-                        ),
-                      ),
-                      IconLabelButton(
-                        onPressed: () => _showProjectGroupDeleteDialog(context),
-                        borderRadius: _buttonBorderRadius,
-                        iconPadding: _buttonIconPadding,
-                        icon: Image.network(
-                          'icons/delete.svg',
-                          width: _iconBoxSide,
-                          height: _iconBoxSide,
-                          fit: BoxFit.contain,
-                          color: theme.accentColor,
-                        ),
-                        label: CommonStrings.delete,
-                        labelStyle: TextStyle(
-                          color: theme.accentColor,
-                        ),
+                      TappableArea(
+                        builder: (context, isHovered, _) {
+                          return IconLabelButton(
```

Comment:
Maybe we can introduce the hover parameters to the IconLabelButton and move the TappableArea logic to the IconLabelButton implementation? Also, I suppose we should think about implementing a similar approach as for MetricsButton. E.g., create the MetricsIconLabelButton and introduce the PositiveIconLabelButton, Negative... and so on.

cc @illia-romanenko @vova-beloded